src/module_library/BioCro.h

```diff
@@ -5,6 +5,7 @@
 #include "AuxBioCro.h"
 
 double resp(double comp, double mrc, double temp);
+double temperature_response(double temp);
```

Comment:
It's confusing when the base file names of the `.h` and `.cpp` files associated with a function are not the same, and I've been trying to standardize these things over time. The `temperature_response` function is implemented in `AuxBioCro.cpp`, so it should be defined in `AuxBioCro.h`, not `BioCro.h`.

Related to this, the function name may be too generic. There are many potential temperature response functions. I would prefer something like `Q10_temperature_response` instead.

Reply:
The Q10 temperature response .

Here's my attempt for a more reusable function. I think the exponential version would be more stable for optimization, since apparently  for floats `pow(x,y)` calculates  `exp(y *  log(x))`  and this way, it isn't have to do `log(x)`. 

```c++
double temperature_response_exponential(
  double temperature, double reference_temperature, double q10_coef
 ){
    double temp_diff = temperature -reference_temperature;
    double u = q10_coef*temp_diff; // q10_coef = log(2)/10
    return exp(  u);
}
```

Reply:
> I think the exponential version would be more stable for optimization, since apparently for floats `pow(x,y)` calculates `exp(y * log(x))` and this way, it isn't have to do `log(x)`.

I usually don't worry about this kind of stuff. The performance difference is likely so small it's not worth spending effort on it.